File: src/main/java/com/bombproj/dto/ProjectMemberDto.java
package com.bombproj.dto;

import lombok.Data;

import java.io.Serializable;

@Data
public class ProjectMemberDto implements Serializable {

    private static final long serialVersionUID = 1L;

    private String userName;// 成员姓名

    private Integer page;

    private String projectId;
    private String createUserId;

}

File: src/main/scala/com/bombproj/dto/ProjectMemberDto.scala
package com.bombproj.dto

import scala.beans.BeanProperty

@SerialVersionUID(1L)
case class ProjectMemberDto() {

    @BeanProperty var userName: String = _ // 成员姓名
    @BeanProperty var page: Int = _
    @BeanProperty var projectId: String = _
    @BeanProperty var createUserId: String = _

}
